sources/packages/backend/apps/db-migrations/src/sql/ApplicationOfferingChangeRequests/Create-application-offering-change-requests.sql

```diff
@@ -0,0 +1,52 @@
+CREATE TABLE sims.application_offering_change_requests (
+  id SERIAL PRIMARY KEY,
+  application_id INT NOT NULL REFERENCES sims.applications(id),
+  change_offering_id INT NOT NULL REFERENCES sims.education_programs_offerings(id),
+  active_offering_id INT NOT NULL REFERENCES sims.education_programs_offerings(id),
+  application_offering_change_request_status sims.application_offering_change_request_status_types NOT NULL,
+  assessed_date TIMESTAMP WITH TIME ZONE,
+  assessed_by INT REFERENCES sims.users(id),
+  student_action_date TIMESTAMP WITH TIME ZONE,
+  reason VARCHAR(500) NOT NULL,
+  ministry_note_id INT REFERENCES sims.notes(id),
```

Comment:
I believe that we never used `ministry_note_id` and we try to create something more close to the context. For this one I would suggest `assessed_note_id`.

Reply:
actually, when I checked another table, we were either using `note_id` or `entity_note_id`, eg, `restrction_note_id`. so I thought if I use `application-offering-change-requests_note_id` if it confuses dev with the reason column. Thats why I gave this name, but the name you suggested also makes sense. I will update